cernopendata_client/cli.py

```diff
@@ -42,6 +42,7 @@
 
 @click.group()
 def cernopendata_client():
+    """Opendata client for interacting with CERN Open Data portal."""
```

Comment:
Command-line client for interacting with CERN Open Data portal.

Reply:
Done 👍